polkadot-parachain/src/command.rs

```diff
@@ -214,6 +217,21 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 			"polkadot-local",
 		)),
 
+		// -- Glutton
+		"glutton-dev" => Box::new(chain_spec::glutton::glutton_development_config(
+			para_id.expect("Must specify parachain id"),
+		)),
+		"glutton-local" => Box::new(chain_spec::glutton::glutton_local_config(
+			para_id.expect("Must specify parachain id"),
+		)),
+		// the chain spec as used for generating the upgrade genesis values
+		"glutton-genesis" => Box::new(chain_spec::glutton::glutton_config(
+			para_id.expect("Must specify parachain id"),
```

Comment:
@Szegoo Did you try to run a collator with this commands? `--chain=glutton-dev`, etc?

I think it will fail unless you modify `fn extract_parachain_id()` to return a `para_id` != `None`.

Reply:
Yes, I have tried running the collator, but I have done it by setting `--chain` to the chain-spec json file of Glutton and it worked that way.

I did not catch this problem, so yes I need to update the code. Good catch.